src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -282,6 +303,30 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
   return (
     <div className="NavBarWrapper">
       <div className={containerClasses}>
+        <div className="NavBarSchedule__breadcrumb">
+          <button
+            onClick={() => setFilterRelatedEvents(false)}
+            className={filterRelatedEvents ? "button--a disabled" : "button--a"}
+          >
+            {sovereignVenue?.name}
+          </button>
+
+          {venueId !== sovereignVenue?.id && (
+            <>
+              {" "}
+              /{" "}
+              <button
+                onClick={() => setFilterRelatedEvents(true)}
+                className={
+                  !filterRelatedEvents ? "button--a disabled" : "button--a"
```

Comment:
Is it actually right to add "button" classname to `<button/>` tag. 
It's worth keeping more sense here

Reply:
if i add the "button" classname it styles it like a button, whereas "button--a" styles it as a link